app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+from pathlib import Path
+import os
+
+
+def move_file(command: str) -> None:
+    comm_list = command.split(" ")
```

Comment:
You can declare all three variables here in one line something like (also don’t use space in split as it use it by default)
```suggestion
    command, old_file, new_file = command.split()
```